dss/operations/checkout.py

```diff
@@ -151,37 +150,43 @@ def process_keys(self):
                 checkout_status[_key] = bundle_internal_status
             elif _key.startswith(FILE_PREFIX):
                 temp = collections.defaultdict(blob_checkout=False, should_be_cached=False)
-                file_metadata = self._verify_get(self.handle, self.replica.bucket, _key)
+                file_metadata = self._get_metadata(self.handle, self.replica.bucket, _key)
+                if not file_metadata:
+                    checkout_status[_key] = 'KEY_DOES_NOT_EXIST'
+                    continue
                 blob_key = compose_blob_key(file_metadata)
-                blob_status = self._verify_blob_existance(self.handle, self.bucket, blob_key)
+                blob_status = self._verify_blob_existance(self.handle, self.checkout_bucket, blob_key)
                 if blob_status:
                     temp['blob_checkout'] = True
-                if cache_flow.should_cache_file(_file['content-type'], _file['size']):
+                if cache_flow.should_cache_file(file_metadata['content-type'], file_metadata['size']):
                     temp['should_be_cached'] = True
 
                 for x in ['name', 'uuid', 'version']:
-                    temp.update({x: _file[x]})
+                    temp.update({x: file_metadata[x]})
                 checkout_status[_key] = collections.defaultdict(uuid=temp)
-        print(json.dumps(checkout_status, sort_keys=True))
+            else:
+                checkout_status[_key] = 'INVALID_KEY_REGEX'
```

Comment:
same comment as above

Reply:
This situation is different from that above in my mind since the key is malformed (and thus never could exist).  Should we raise to warn/disallow the user?

Reply:
https://github.com/HumanCellAtlas/data-store/pull/2549#discussion_r340363942 from the wizard himself. :male_mage